TESTS/bug_79_double_complex.f

```diff
@@ -217,7 +217,12 @@ program bug_79_double_complex
             go to 9000
          end if
  9000 continue
-      if (res1 .ne. res2) then
+c     Compare difference to double precision 0 instead of using
+c     bitwise comparison operator .ne.
+      if (abs(res1 - res2) > 0.0D+0) then
+         write(6,'(a,3e24.16)')
```

Comment:
`write(6,'(a,e24.16)')` should be enough if you print out only `res1-res2`

Reply:
Do you think it would be better to print `res1` and `res2` individually too? Then I'll modify accordingly.

Reply:
Yes ! Anyway, do what you think the most relevant :D